core/src/main/scala/io/qbeast/core/transform/Transformer.scala

```diff
@@ -147,8 +147,8 @@ object ColumnStats {
     val min = stats("min")
     val max = stats("max")
     val count = stats("count").toLong
-    val stddev = stats("stddev").toDouble
-    val mean = stats("mean").toDouble
+    val stddev = stats("stddev")
```

Comment:
If we are not using stddev, we should not calculate it. we should avoid using all metrics that are not useful for us.